rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import copy from 'rollup-plugin-copy';
+import del from 'rollup-plugin-delete';
+import dynamicImportVars from '@rollup/plugin-dynamic-import-vars';
+import glob from 'glob-all';
+import resolve from '@rollup/plugin-node-resolve';
+
+const jsGlob = [
+	'@(components|controllers|directives|helpers|mixins|templates)/**/*.js',
+	'!**/*@(test|axe|visual-diff).js',
+];
+const nonJsGlob = [
+	'@(components|controllers|directives|helpers|mixins|templates)/**/*.*',
+	'*.*',
+	'!**/*.@(js|md|json)',
+	'!**/screenshots/**/*',
+];
```

Comment:
This selects all non-JS files in our "main" directories, also filtering out Markdown, JSON, and screenshots. This saves us a lot of MB (from 50.7MB down to 5.67MB!).

I confirmed that there are no cases where any of the above files are being imported in HTML or JS.